newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -380,34 +380,31 @@ common: &default_settings
     #label_name: label_value
 
   security:
-    # Default is false
-    enabled: true
+    # Determines whether the csec security instrumentation modules execute or not. Default is false.
+    enabled: false
 
     # NR security provides two modes IAST and RASP
-    # Default is IAST
-    # mode: IAST
+    # Default is IAST. Due to the invasive nature of IAST scanning, DO NOT enable this mode in either a
+    # production environment or an environment where production data is processed.
+    mode: IAST
 
     # New Relic’s SaaS connection URLs
-    # validator_service_url: wss://csec.nr-data.net
-
-    # To completely disable security
-    # set agent.enabled flag to false
-    # If the flag is set to false,
-    # the security module is not loaded
-    # This property is read only once at
-    # application start.
-    # agent:
-    #   enabled: true
-
-    # Following category of security events
-    # can be disabled from generating.
-    # detection:
-    #   rci:
-    #     enabled: true
-    #   rxss:
-    #     enabled: true
-    #   deserialization:
-    #     enabled: true
+    validator_service_url: wss://csec.nr-data.net
+
+    # To completely disable csec security, set agent.enabled flag to false. If the flag is set to false,
+    # the security module is not loaded. This property is read only once at application start. Default is
+    # false.
+    agent:
+      enabled: false
+
+    # These are the category of csec security events that can be disabled from generating.
+    detection:
+      rci:
+        enabled: true
+      rxss:
+        enabled: true
+    deserialization:
```

Comment:
I believe `deserialization` should be indented under the `detection` stanza, not on the same level as it.

Reply:
Thank you - corrected!